lstchain/image/muon/muon_analysis.py

```diff
@@ -131,90 +129,135 @@ def analyze_muon_event(event_id, image, geom, equivalent_focal_length,
     tailcuts = [10, 5]
 
     cam_rad = 2.26 * u.deg
-    min_pix = 148  # 8%
 
+    # some cuts for good ring selection:
+    min_pix = 148                              # (8%) minimum number of pixels in the ring with >0 signal
+    min_pix_fraction_after_cleaning = 0.1      # minimum fraction of the ring pixels that must be above tailcuts[0] 
+    min_ring_radius = 0.8*u.deg                # minimum ring radius
+    max_ring_radius = 1.5*u.deg                # maximum ring radius 
+    max_radial_stdev = 0.1*u.deg               # maximum standard deviation of the light distribution along ring radius
+    max_radial_excess_kurtosis = 1.            # maximum excess kurtosis
+    min_impact_parameter = 0.2                 # in fraction of mirror radius
+    max_impact_parameter = 0.9                 # in fraction of mirror radius
+    ring_integration_width = 0.4               # +/- integration range along ring radius, in fraction of ring radius
+    outer_ring_width = 0.2                     # in fraction of ring radius, width of ring just outside the integrated muon ring, used to check pedestal bias
```

Comment:
is this condition implying:
ring_width < ring_radius * outer_ring_width ?

Reply:
then I do not understand the `outer` part of the name

Reply:
> then I do not understand the `outer` part of the name

"outer" refers to the fact that this is a ring _just outside_ the muon ring in which the muon-related charge is integrated.

Reply:
Are hard coded cuts inline in the code really a good idea?

Reply:
> Are hard coded cuts inline in the code really a good idea?

Certainly to be improved... but I'd prefer to close this first (hard-coded cuts were there already,  I just added some, and tried to place them together for clarity)

Reply:
ok